microprofile.jdt/com.redhat.microprofile.jdt.core/plugin.properties

```diff
@@ -10,4 +10,5 @@
 ###############################################################################
 pluginName=JDT MicroProfile Extension
 providerName=Red Hat
-propertiesProviders.name=MicroProfile properties providers extension
\ No newline at end of file
+propertiesProviders.name=MicroProfile properties providers extension
+projectType.name=Project types extension
```

Comment:
As you use `%projectTypes.name` in the plugin.xml it should be `projectTypes.name`